lib.php

```diff
@@ -390,8 +402,61 @@ function coursework_grade_item_update($coursework, $grades = null) {
         $grades = null;
     }
 
-    return grade_update('mod/coursework', $courseid, 'mod', 'coursework', $coursework->id, 0,
-                        $grades, $params);
+    return grade_update(
+        'mod/coursework', $courseid, 'mod', 'coursework',
+        $coursework->id, 0, $grades, $params
+    );
```

Comment:
one per line and trailing comma?

Reply:
We could do but I don't think there is a rule for that - there is if we are defining an array but here we are calling a function